subscription_integration_test.go

```diff
@@ -1012,3 +1013,26 @@ func TestSubscriptionTransactions(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// In order to run this test you have to create a test subscription
+// that has a status of "Past Due". You can get such a subscription
+// by creating one with a trial period of one day and a price of 2000
+// USD (this amount will fail in the sandbox environment). After the
+// trial expires, the subscription will have the desired "Past Due"
+// status. Then you can try to charge an amount lower than 2000 which
+// then should work when we retry the charge.
+func TestSubscriptionRetryCharge(t *testing.T) {
+	t.Skip("Needs manual setup")
```

Comment:
I'm going to investigate if we can avoid manual setup for this test. Thanks for including a test!!!

Reply:
Where I work we couldn't find a way. If you find one please let us know! :)